src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/HttpSamDAO.scala

```diff
@@ -27,7 +28,7 @@ class HttpSamDAO( implicit val system: ActorSystem, implicit val executionContex
     val pipeline = sendReceive
     pipeline(Get(samStatusUrl)) map { response =>
       val ok = response.status.isSuccess
-      SubsystemStatus(ok, if (ok) Option(List(response.entity.asString)) else None)
+      SubsystemStatus(ok, if (!ok) Option(List(response.entity.asString)) else None)
```

Comment:
I caught this during development: previously, if Sam returned an OK status, we'd display messages, but if it returned not-OK, we'd suppress them. That's backwards.

Reply:
Nice catch!